app/models/accounting/transaction.rb

```diff
@@ -84,6 +84,20 @@ class Accounting::Transaction < ApplicationRecord
     order(:txn_date, "options.position", :created_at)
   }
   scope :interest_type, -> { where(loan_transaction_type_value: LOAN_INTEREST_TYPE) }
```

Comment:
should I replace usages of the  :interest_type scope with new :by_type('interest') scope?

Reply:
You can replace the internal implementation so it's `scope :interest_type, -> { by_type(LOAN_INTEREST_TYPE) }`